### PR TITLE
feat: update coverage plugin with latest components

### PR DESCRIPTION
**Pull request by snitin315**
## Description

Added/Updated following components - 

- Accordion 
- Drawer
- Menu 
- Step Group 
- Top Nav
- FileUpload
- ButtonGroup

image

### CODE REVIEW

File: packages/plugin-figma-blade-coverage/README.md [hunk 1]
```diff
@@ -81,6 +76,32 @@ It excludes the following nodes from coverage calculation:
 
 <br/>
 
+## Local Development
```
Comment (saurabhdaware) [inline packages/plugin-figma-blade-coverage/README.md:79]: Nice

File: packages/plugin-figma-blade-coverage/src/bladeLibraryConstants.ts [hunk 2]
```diff
@@ -58,13 +64,45 @@ export const bladeThemeData = {
...
+    'MenuItem/Trailing': {
+      key: '9c7a7356118794224990529e4922d8ec89a77b56',
+    },
+    'MenuItem/Leading': {
+      key: 'cbeaaaeb4fbf0a830a97687bf50c50aa2cfe8b2a',
+    },
+    L2SubMenuItem: {
+      key: '06cc9d71153af14dc0ec0cb22835596420fa7c93',
+    },
+    L1SubMenuItem: {
```
Comment (saurabhdaware) [inline packages/plugin-figma-blade-coverage/src/bladeLibraryConstants.ts:103]: which components are these?

**Review by saurabhdaware — APPROVED** [review]
Looks good to me. One doubt below

File: packages/plugin-figma-blade-coverage/src/bladeLibraryConstants.ts (hunk 2 above)
Comment (snitin315) [reply packages/plugin-figma-blade-coverage/src/bladeLibraryConstants.ts:103]: [Here](https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=91761-122151&t=6xCmJLzuk5IdLvuB-11)

**Review by saurabhdaware — APPROVED** [review]

**Pull request merged**